platform/ui/src/components/SplitButtonToolbar/SplitButtonToolbar.tsx

```diff
@@ -0,0 +1,192 @@
+import Tooltip from '../Tooltip';
+import ListMenu from '../ListMenu';
+import ToolbarButton from '../ToolbarButton';
+
+const baseClasses = {
+  Button: 'flex items-center rounded-md border-transparent cursor-pointer group/button',
+  Primary:
+    'h-full border-l-2 border-t-2 border-b-2 rounded-tl-md rounded-bl-md group/primary !pl-2 !py-2',
+  Secondary:
+    'h-full flex items-center justify-center rounded-tr-md rounded-br-md w-4 border-2 border-transparent group/secondary',
+  SecondaryIcon: 'w-4 h-full stroke-1',
+  Separator: 'border-l py-2.5',
+  Content: 'absolute z-10 top-0 mt-12',
+};
+
+const classes = {
+  Button: ({ isExpanded }) =>
+    classNames(
+      baseClasses.Button,
+      !isExpanded && 'hover:!bg-primary-dark hover:border-primary-dark'
+    ),
+  Interface: 'h-full flex flex-row items-center',
+  Primary: ({ isExpanded, isActive }) =>
+    classNames(
+      baseClasses.Primary,
+      isActive
+        ? isExpanded
+          ? 'border-primary-dark !bg-primary-dark hover:border-primary-dark !text-primary-light'
+          : 'border-primary-light bg-primary-light border-2 rounded-md !p-2'
+        : `focus:!text-black focus:!rounded-md focus:!border-primary-light focus:!bg-primary-light ${isExpanded ? 'border-primary-dark bg-primary-dark !text-primary-light' : 'border-secondary-dark bg-secondary-dark group-hover/button:border-primary-dark group-hover/button:text-primary-light hover:!bg-primary-dark hover:border-primary-dark focus:!text-black'}`
+    ),
+  Secondary: ({ isExpanded, primary }) =>
+    classNames(
+      baseClasses.Secondary,
+      isExpanded
+        ? 'bg-primary-light !rounded-tr-md !rounded-br-md'
+        : primary.isActive
+          ? 'bg-secondary-dark'
+          : 'hover:bg-primary-dark bg-secondary-dark group-hover/button:border-primary-dark'
+    ),
+  SecondaryIcon: ({ isExpanded }) =>
+    classNames(
+      baseClasses.SecondaryIcon,
+      isExpanded
+        ? 'text-primary-dark'
+        : 'text-primary-active group-hover/secondary:text-primary-light'
+    ),
+  Separator: ({ primary, isExpanded, isHovering }) =>
+    classNames(
+      baseClasses.Separator,
+      isHovering || isExpanded || primary.isActive ? 'border-transparent' : 'border-primary-active'
+    ),
+  Content: ({ isExpanded }) => classNames(baseClasses.Content, isExpanded ? 'block' : 'hidden'),
+};
+
+const DefaultListItemRenderer = props => {
+  const { t, icon, label, className, isActive } = props;
+  return (
+    <div
+      className={classNames(
+        'flex h-8 w-full flex-row items-center p-3',
+        'whitespace-pre text-base',
+        className,
+        `${isActive ? 'hover:opacity-80' : 'hover:bg-primary-dark '}`
+      )}
+    >
+      {icon && (
+        <span className="mr-4">
+          <Icon
+            name={icon}
+            className="h-5 w-5"
+          />
+        </span>
+      )}
+      <span className="mr-5">{t?.(label)}</span>
+    </div>
+  );
+};
+
+const SplitButtonToolbar = ({
```

Comment:
How is this different from split button, or is it just the naming?  This needs at least a bit of documentation to explain what it is here.